arxiv_public_data/fulltext.py

```diff
@@ -173,8 +173,9 @@ def fulltext(pdffile: str, timelimit: int=TIMELIMIT):
     output = fixunicode.fix_unicode(output)
     #output = stamp.remove_stamp(output, split=STAMP_SEARCH_LIMIT)
     wordlength = average_word_length(output)
-
+    
```

Comment:
please remove trailing whitespace (consider running it through pylint to identify these things)

Reply:
OK done!